Sources/Workspace/Workspace.swift

```diff
@@ -251,6 +254,12 @@ public class Workspace {
     /// The manifest loader to use.
     let manifestLoader: ManifestLoaderProtocol
 
+    /// The tools version currently in use.
+    let currentToolsVersion: ToolsVersion
```

Comment:
When would this be different than the static one in ToolsVersion?  In particular, does this need to be a property here if it is always the same for any given set of tools?